packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.scss

```diff
@@ -39,6 +39,39 @@
     top: 0;
     z-index: 100;
     background: var(--phoenix-background-color-secondary);
+
+    .head-wrapper {
```

Comment:
Then I would suggest to replace the entire styles inside with this:

```scss
.head-wrapper {
  display: flex;
  align-items: center;

  .sort-options {
    display: flex;
    flex-direction: row;

    .icon-wrapper {
      display: flex;
      width: 1rem;
      height: 1rem;
      padding: 0.2rem;

      &.up {
        transform: rotate(180deg);
      }

      svg {
        width: 100%;
        height: 100%;
      }
    }
  }
}
```

This makes it a bit simpler and cleaner.

![image](https://user-images.githubusercontent.com/36920441/143717408-34629aae-6327-4e56-a5a9-1ab79a7f1b43.png)